src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -622,8 +637,25 @@ private void OnTestRunAbort(IInternalTestRunEventsHandler testRunEventsHandler,
         SetOperationComplete();
 
         var reason = GetAbortErrorMessage(exception, getClientError);
-        EqtTrace.Error("TestRequestSender: Aborting test run because {0}", reason);
-        LogErrorMessage(string.Format(CultureInfo.CurrentCulture, CommonResources.AbortedTestRun, reason));
+        // If it isn't a generic reason then, log it dispite if blame enabled or no.
+        if (reason != "Test host process crashed")
+        {
+            EqtTrace.Error("TestRequestSender: Aborting test run because {0}", reason);
+            LogErrorMessage(string.Format(CultureInfo.CurrentCulture, CommonResources.AbortedTestRun, reason));
+
+        }
+        // If it's a generic reason if blame not enabled ask the user to enable it for more details. 
+        else if (!BlameIsEnabled())
+        {
+            EqtTrace.Error("TestRequestSender: Aborting test run");
+            LogErrorMessage(string.Format(CultureInfo.CurrentCulture, "The active test run was aborted. Use blame for more details."));
```

Comment:
You should always add new messages into the resx so that entries gets translated (here and in the else below).